integrationTests/chainSimulator/bridge/executeAndBurn_test.go

```diff
@@ -151,6 +168,7 @@ func simulateExecutionAndDeposit(
 		NumNodesWaitingListShard: 0,
 		AlterConfigsFunction: func(cfg *config.Configs) {
 			cfg.GeneralConfig.VirtualMachine.Execution.TransferAndExecuteByUserAddresses = []string{whiteListedAddress}
+			cfg.EpochConfig.EnableEpochs.DynamicESDTEnableEpoch = 3
```

Comment:
Why not 0 and test it with dynamic nfts?

Reply:
The Epoch number I don't think it matters, I generate blocks for 3 epochs in the beginning.
Unfortunately, I can't set Dynamic ESDTs because rust framework is not updated with all out ESDTTypes and gives me an error.

Reply:
Does esdt prefix logic work with this flag enabled?